extensions/amp-story/1.0/page-advancement.js

```diff
@@ -524,14 +524,19 @@ export class ManualAdvancement extends AdvancementConfig {
         tagName = el.tagName.toLowerCase();
 
         if (
-          tagName === 'amp-story-cta-layer' ||
           tagName === 'amp-story-page-attachment' ||
           tagName === 'amp-story-page-outlink'
         ) {
           valid = false;
           return false;
         }
 
+        if (tagName === 'amp-story-cta-layer') {
+          const anchorSet = el.querySelectorAll('a');
+          valid = anchorSet && anchorSet.length > 1;
+          return valid;
+        }
```

Comment:
All cta-layers with 1 link will be converted and no longer exist, leaving you with the ones with more than a link. You should be able to remove this block and things would work as expected! Deleting the line above is enough.